source/api_documentation/network_integration/call_activity_tracking/index.rst

```diff
@@ -0,0 +1,51 @@
+Call Activity Tracking
+======================
+
+Manage call activity tracking for campaigns
+"""""""""""""""""""""""""""""""""""""""""""
+
+Call activity tracking can be set for a campaign to determine whether or not call activity tracking should be enabled. The call activity will be tracked if the full condition is met.
+
+Set Call Activity Tracking
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To conditionally enable call activity tracking, set campaign's owning network `call_activity_access` feature flag to `true`, and set ``"call_activity_tracking"`` to ``true``.
+
+POST
+
+``https://invoca.net/api/@@CAMPAIGN_FEATURES_API_VERSION/<network_id>/advertisers/<advertiser_id_from_network>/advertiser_campaigns/<advertiser_campaign_id_from_network>.json``
+
+Example Request Body
+
+.. code-block:: json
+
+  {
+    "ivr_tree": {
+      "call_activity_tracking": true,
+      "root": {
+        "node_type": "Connect",
+        "destination_phone_number": "8056173768",
+        "destination_country_code": ""
+      }
+    }
+  }
+
+Conditions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. list-table::
+  :widths: 30 50 30
+  :header-rows: 1
+  :class: parameters
+
+  * - Boolean
```

Comment:
Here we can change this to be `Option` maybe? And then list the three different call [tracking options](https://github.com/Invoca/web/blob/7f4d29c4c7cd8f04066f7f7e6556e2c0777ceef1/app/models/network.rb#L27-L28) instead

Reply:
Fixed, thank you!